src/utils/config-vars.js

```diff
@@ -68,7 +70,7 @@ class ConfigVars {
 
         await this.checkPrepTarg();
 
-        this.fromConfigFile(this.result.PLAYCANVAS_TARGET_DIR, TARGET_CONFIG_FILE);
+        this.fromTargetConfigFile(this.result.PLAYCANVAS_TARGET_DIR, TARGET_CONFIG_FILE);
```

Comment:
This is great!

I'm just thinking, it might be clearer for the user to specify the config filename directly (instead of a profile name).

Then the only code change here would be to use that filename instead of TARGET_CONFIG_FILE.

This way, the user knows exactly which file is being referenced and they can name the file however they like.

What do you think @figo2264?

Reply:
Basically, the main idea is to allow multiple config file on a single project and I adopted the concept from spring profile.

So, what you meant is just explicitly point out the config file name like what we normally do in typescript config, right?

Reply:
Yes. I'm imagining something like:

`--config='pcconfig-dev.json'`

Which would load settings from 'pcconfig-dev.json' instead of 'pcconfig.json'.